core/src/main/java/com/graphhopper/routing/profiles/BitEncodedValue.java

```diff
@@ -0,0 +1,54 @@
+package com.graphhopper.routing.profiles;
+
+/**
+ * This class provides easy access to just one bit.
+ */
+public final class BitEncodedValue extends IntEncodedValue {
```

Comment:
Nice, GH missed a Boolean Encoded Value, not sure if we should call it like this instead of Bit?